assets/Mods/shared/Types.lua

```diff
@@ -819,4 +819,66 @@ function RemoteUnrealParam:type() end
 
 ---@class TSoftClassPtr<T> : string
 ---@class TSoftObjectPtr<T> : string
----@class TSubclassOf<T> : UClass
\ No newline at end of file
+---@class TSubclassOf<T> : UClass
+
+
+---@class UEnum
+
+UEunm = {}
+
+--- Returns the `FName` that corresponds to the specified value.
+---@param Value integer
+function UEunm:GetNameByValue(Value) end
```

Comment:
All the instances of `UEnum` are typo'd as `UEunm` in this file.

Reply:
Fixed.